pool/session.go

```diff
@@ -0,0 +1,19 @@
+package pool
+
+import (
+	"context"
+
+	"github.com/nspcc-dev/neofs-sdk-go/client"
+)
+
+// SessionCreate opens a session with the node server on the remote endpoint.
+//
+// See details in [client.Client.SessionCreate].
+func (p *Pool) SessionCreate(ctx context.Context, prm client.PrmSessionCreate) (*client.ResSessionCreate, error) {
```

Comment:
how do we handle sessions... with different nodes? what a user should expect when it opens a session with... pool?

Reply:
Can't be provided, just like `EndpointInfo`.